python/ray/serve/http_proxy.py

```diff
@@ -116,19 +99,19 @@ def _make_error_sender(self, scope, receive, send):
         assert scope["type"] == "http"
         current_path = scope["path"]
         if current_path == "/-/routes":
-            await Response(self.route_table_cache).send(scope, receive, send)
+            await Response(self.route_table).send(scope, receive, send)
             return
 
```

Comment:
send error here when `self.route_table` is `self.router_handle` is None

Reply:
Good idea. Will do.